### PR TITLE
Update `eu_dcc_remove` to reflect latest app changes

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR applies updates to the FAQ entry:
- https://www.coronawarn.app/de/faq/#eu_dcc_remove
- https://www.coronawarn.app/en/faq/#eu_dcc_remove

## What was changed?

- All occurrences of "Remove" in this FAQ entry were replaced with "Delete" (to reflect a change in the wording used by the app)
- All occurrences of "Entfernen" in this FAQ entry were replaced with "Löschen" (to reflect a change in the wording used by the app)
- 2 new sentences reg. the recycle bin feature were added
- "Recovery Certificate" was changed to "Certificate of Recovery", to reflect how the certificate is called in the app
- Minor improvements were done to the English text

## Screenshots

| English | German |
|---|---|
| Bildschirmfoto 2021-12-30 um 00 46 12 | Bildschirmfoto 2021-12-30 um 00 45 15 |

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -198,12 +198,12 @@
...
+                            "Mithilfe dieser Schritt-für-Schritt Anleitung können Sie Zertifikate aus der Corona-Warn-App löschen. Bitte beachten Sie, dass die ersten drei Schritte für sowohl iOS als auch Android gelten, aber nur jeweils einer der letzten beiden Schritte befolgt werden muss, je nachdem welches Betriebssystem mit Ihrem übereinstimmt. Impfzertifikate, Testzertifikate und Genesenenzertifikate werden als 'Zertifikate' zusammengefasst, da der Ablauf bei allen identisch ist.<br>Nachdem Sie die nachfolgenden Schritte ausgeführt haben, wird das Zertifikat vorerst in den Papierkorb verschoben. Dort wird es dann nach 30 Tagen automatisch gelöscht, außer Sie stellen es von dort wieder her oder löschen es endgültig (Die Papierkorb-Funktion ist ab Version 2.13 verfügbar).",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:205]: "... oder löschen es endgültig (Die Papierkorb-Funktion ..."

There should be a full-stop "." after "endgültig".

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Missing punctuation mark.

File: src/data/faq_de.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Thanks Mike, I fixed the punctuation according to https://www.duden.de/sprachwissen/sprachratgeber/Klammern-in-Verbindung-mit-Komma-oder-Punkt.

File: src/data/faq.json [hunk 2]
```diff
@@ -201,7 +201,7 @@
...
+                            "Using this step-by-step guide, you can delete certificates from the Corona-Warn-App. Please note that the first three steps apply to both iOS and Android, but only one of the last two steps needs to be followed, depending on which operating system matches yours. Vaccination certificates, test certificates and certificates of recovery are all called 'Certificates' in the following, as the process of removing them is identical.<br>After you have performed the following steps, the certificate is moved to the recycle bin for the time being. There, it will then be automatically deleted after 30 days, unless you restore it from there or permanently delete it. (the recycle bin feature is available starting with version 2.13.)",
```
Comment (MikeMcC399) [inline src/data/faq.json:204]: @Ein-Tim 
If you add a full stop after "it", then you need to capitalize "The" in  "The recycle bin ..." which follows.

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq_de.json]: The German looks correct. The English is now wrong.

File: src/data/faq.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq.json]: @MikeMcC399 
Thank you!
Fixed with [`09b3dbb`](https://github.com/corona-warn-app/cwa-website/pull/2262/commits/09b3dbbc41229a0308372ae1161536859b5371ab).

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 LGTM

**Review by MikeMcC399 — APPROVED** [review]
The punctuation now looks correct.

**Pull request merged**